writing/kill-the-dog.md

```diff
@@ -0,0 +1,52 @@
+# Killing the Dog - My new reason for practicing mindfulness
+
+- Theory - Learning to slow down, and be present will have massive benefits when get old and thinking becomes harder. If you've made it a habit
+
```

Comment:
I like this.... I think you added it today maybe, don't remember seeing it previously.  Interesting and unique twist.  Habitualizing mindfulness.